packages/orbit-components/src/Card/CardSection/index.tsx

```diff
@@ -105,42 +121,45 @@ export default function CardSection({
           {actions && <Actions actions={actions} />}
         </div>
       )}
-
       {(title != null || header != null) && !expandable && (
```

Comment:
I am a bit unsure about the focus order in this case (when both `onClick` and `actions` are provided), first the action button is being focused and then the card section is being focused, shouldn't it be the other way around? Same as when it's expandable? 🤔